variables.tf

```diff
@@ -15,6 +15,17 @@ variable "force_destroy" {
   default     = false
 }
 
+variable "grants" {
+  description = "An ACL policy grant. Conflicts with `acl`, which must be set to `null`"
```

Comment:
The description should note that it's a list. It currently suggests that it is a single grant configuratino

Reply:
Gotcha. Updated the phrasing.